### ISSUE TITLE
Show a toggle/hover to include the commit message info that is shown on the Commits tab inline in the diff

### ISSUE THREAD

**Issue by robertoandrade** [Yatser/prettypullrequests#31]
Some times you need an explanation as to why that line of code was changed and the explanation is in the commit message but switching from one view to the other to find the exact commit that diff relates to just to read what the commit message was sucks... Almost like a blame type feature on the PR but including the message.

**Comment by bnetter** [comment]
:+1:

I actually asked Github for this a while back. Got this answer:

```
Hi Benjamin,

Thanks for the feedback. We'll keep this in mind for consideration in future.

Thanks,
James
```

**Comment by robertoandrade** [comment]
Yeah I guess that's why things like PrettyPullRequests and Zenhub are popular extensions these days :)

**Comment by Yatser** [comment]
@robertoandrade @bnetter my former coworker talked to some github employees at Pycon in Montreal last year. She told them about this plugin and asked them about some other new (very basic) features she would like to see in github. Their response was to submit those requests to me haha.

I was REALLY hoping this would be a temporary plugin solving a temporary solution, which is why I haven't been adding any and every feature to it that people have requested, but it seems they want to keep the PR review to a very simple tool and rely on extensions to add functionality (they even officially link to these extensions). So I think I might have to continue to support and grow this extensions features for quite some time, since there is an active weekly userbase of 1800 users and growing.
